app/views/devise/registrations/edit.html.erb

```diff
@@ -1,43 +1,54 @@
-  <% end %>
-
-  <div class="field">
-    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-    <% if @minimum_password_length %>
-      <br />
-      <em><%= @minimum_password_length %> characters minimum</em>
-    <% end %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "off" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
-    <%= f.password_field :current_password, autocomplete: "off" %>
-  </div>
-
-  <div class="actions">
-    <%= f.submit "Update" %>
-  </div>
-<% end %>
-
-<h3>Cancel my account</h3>
-
-<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
-
-<%= link_to "Back", :back %>
+<%= render 'layouts/navbar'%>
+<div class="ui container">
+  <div class="ui segments">
+    <div class="ui segment orange inverted">
+      <h3 align="center">
+        Edit Profile of <%= resource_name.to_s.humanize %>
+      </h3>
+    </div>
+    <div class="ui segment">
+      <div class="description">
+
+        <%= form_for(resource, as: resource_name, :class => 'ui form segment', url: registration_path(resource_name), html: { method: :put }) do |f| %>
+        <%= devise_error_messages! %>
+      </div>
+      <div class="ui divider"></div>
+      <div class='ui form segment'>
+
+
+
+          <%= f.label :email %><br />
+          <%= f.email_field :email, autofocus: true %>
+          <br/>
+
+        <div class="ui divider hidden"></div>
+
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+        <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+        <% end %>
+
+        <%= f.label :password %> 
+        <%= f.password_field :password, autocomplete: "off" %><br/>
+        <i> (leave blank if you don't want to change it) 
+          <% if @minimum_password_length %></i>
+          <em><%= @minimum_password_length %> characters minimum</em>
+          <% end %>
+          <br/><br/>
+          <%= f.label :password_confirmation %><p>
+          <%= f.password_field :password_confirmation, autocomplete: "off" %>
+          <br/><br/>
+          <%= f.label :current_password %> 
+          <%= f.password_field :current_password, autocomplete: "off"%>
+          <br/>
+
+          <div class="actions">
+            <%= f.submit "Update" , :class=>"ui primary submit labeled icon button" %>
+          </div>
+          <% end %>
+
+          <h3>Would you like to cancel your account?</h3>
+          <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete, :class=>"ui primary submit labeled icon button" %></p>
```

Comment:
Stick to a consistent hash declaration. Use `key: value` instead of `:key => value`